packages/@aws-cdk/aws-kms/lib/key.ts

```diff
@@ -365,12 +365,26 @@ export class Key extends KeyBase {
   constructor(scope: Construct, id: string, props: KeyProps = {}) {
     super(scope, id);
 
-    this.policy = props.policy || new iam.PolicyDocument();
-    this.trustAccountIdentities = props.trustAccountIdentities || false;
-    if (this.trustAccountIdentities) {
-      this.allowAccountIdentitiesToControl();
+    const defaultKeyPoliciesEnabled = FeatureFlags.of(this).isEnabled(cxapi.KMS_DEFAULT_KEY_POLICIES);
```

Comment:
I would rename this to `defaultKeyPoliciesFeatureFlagEnabled`, just to make it absolutely clear in the below code what it's checking.

Reply:
Did you commit the `cxapi` file? I'm not seeing it in this diff.